functions/Restore-DbaDatabase.ps1

```diff
@@ -221,8 +221,30 @@ folder for those file types as defined on the target instance.
     }
     PROCESS
     {
-        
-        foreach ($f in $path)
+		$base = $SqlServer.Split("\")[0]
+		
+		if ($base -eq "." -or $base -eq "localhost" -or $base -eq $env:computername -or $base -eq "127.0.0.1")
+		{
+			# they can use whatever path they like
+		}
```

Comment:
Move the base processing back into begin.
store whether it's local or not in
$IsLocal
Will not catch IPv6 localhost